src/templates/staff/assignment.html

```diff
@@ -285,14 +326,38 @@ <h5 class="modal-title" id="mdl-edit-assn-label">Edit assignment</h5>
                                         </form>
                                     </div>
                                     <div class="modal-footer">
-                                        <button type="submit" form="edit-assn" class="btn btn-primary" id="mdl-edit-assn-save">
+                                        <button type="button" class="btn btn-outline-danger w-100" data-toggle="modal" data-target="#delete-confirmation">
+                                            <span class="fa fa-trash-alt" style="margin-right: 0.2em;"></span>
+                                            Delete
+                                        </button>
+                                        <button type="submit" form="edit-assn" class="btn btn-primary w-100" id="mdl-edit-assn-save">
                                             <span class="fa fa-cog loader" style="margin-right: 0.2em;"></span>
                                             Save
                                         </button>
                                     </div>
                                 </div>
                             </div>
                         </div>
+                        <div class="modal fade" tabindex="-1" role="dialog" id="delete-confirmation">
+                            <div class="modal-dialog p-3" role="document" style="margin-top: 5rem;">
+                                <div class="modal-content">
+                                    <div class="modal-header">
+                                        <h5 class="modal-title">Delete {{ course.name }}</h5>
```

Comment:
> Delete {{ course.name }}

This might be a bit misleading since we're not deleting the course. Maybe `Delete {{assignment.assignment_id}}` here would be more clear?

Reply:
Oops nice catch!